server/src/assets/game/script-game.js

```diff
@@ -849,13 +861,13 @@ function initialise() {
 	});
 
 	// Set the possible destinations for the SWTbahn platform.
-	allPossibleDestinations = allPossibleDestinationsSwtbahnFull;
-//	allPossibleDestinations = allPossibleDestinationsSwtbahnStandard;
+//	allPossibleDestinations = allPossibleDestinationsSwtbahnFull;
+	allPossibleDestinations = allPossibleDestinationsSwtbahnStandard;
 //	allPossibleDestinations = allPossibleDestinationsSwtbahnUltraloop;
 	disableAllDestinationButtons();
 
 	// Set the signal to flag mapping.
-	signalFlagMap = signalFlagMapSwtbahnFull;
+	signalFlagMap = signalFlagMapSwtbahnStandard;
```

Comment:
this overwrites the dynamic setting of allPossibleDestinations earlier.   
The flag maps probably also have to be made dynamic, or not?